Doc/library/profile.rst

```diff
@@ -299,6 +299,13 @@ functions:
       Create a :class:`~pstats.Stats` object based on the current
       profile and print the results to stdout.
 
+      The *sort* parameter specifies the sorting order of the displayed
+      statistics. It accepts tuple of SortKey or accepts a tuple of SortKeys
+      to enable multi-level sorting.
```

Comment:
```suggestion
      statistics. It accepts a single key or a tuple of keys to enable
      multi-level sorting. A key can be either an integer, a string, or a
      SortKey enum identifying the basis of a sort (for example: -1,
      'time', 'name', SortKey.TIME, or SortKey.NAME).
```

Reply:
This one uses a more apparent structure like the [sort_stats doc](https://docs.python.org/3.13/library/profile.html#pstats.Stats.sort_stats).

Reply:
Thanks @egeakman !